recipe/bld.bat

```diff
@@ -0,0 +1,20 @@
+mkdir build
+cd build
+
+cmake ^
+    -G "NMake Makefiles" ^
+    -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
```

Comment:
It seems that this variable is only defined for Win32 systems (according to https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html)